joomla/joomla-cms
# Clean up the javascript inline mess

Pull Request for Issue #NULL .
### Summary of Changes
- XTD buttons SHOULD NOT spit inline javascript!
### Testing Instructions

Set editor to none
- Edit an article and try ALL the xtd-Buttons (the ones under the editor)
- Create menu items for article, contact, selecting an article and a contact should be ok

Repeat the procedure with the other editors: tinyMCE and Codemirror

Test with any 3rd PD editor
### Documentation Changes Required
### B/C considerations

There shouldn't be a break in B/C.

Calling @Fedik, @andrepereiradasilva @rdeutz @wilsonge and @mbabker
